### PR TITLE
Added more information on creating new starter bots.

### PR DESCRIPTION
**Pull request by Jana-Wessels**
Updated the readme for the starterbots to contain more information on creating a bot in a new language. Waiting on confirmation of the final date bots can be submitted and more information on creating Docker containers.

### CODE REVIEW

File: starter-bots/README.md [hunk 1]
```diff
@@ -10,5 +10,55 @@ Starter bots are the bare essentials you need to get going with very little deci
 
 The reference bot on the other hand is capable of playing a game from start to finish with some cleverness built in. This is there to help contestants who want something a bit smarter to work from.
 
+## Additional languages
+
+For any additional languages apart from the initial 4 that have been provided we will be relying on the community to contribute. 
+
+To submt a new language for the tournament the following will need to be provided :
```
Comment (Blaarkies) [inline starter-bots/README.md]:
```suggestion
To submit a new language for the tournament the following will need to be provided :
```

File: starter-bots/README.md [hunk 2]
```diff
@@ -10,5 +10,55 @@ Starter bots are the bare essentials you need to get going with very little deci
 
 The reference bot on the other hand is capable of playing a game from start to finish with some cleverness built in. This is there to help contestants who want something a bit smarter to work from.
 
+## Additional languages
+
+For any additional languages apart from the initial 4 that have been provided we will be relying on the community to contribute.
```
Comment (Blaarkies) [inline starter-bots/README.md]:
```suggestion
For any additional languages (except C#, Java, Javascript and Python 3) we will be relying on the community to contribute. 
```

Comment (Jana-Wessels) [reply starter-bots/README.md]: Updated

File: starter-bots/README.md (hunk 1 above)
Comment (Jana-Wessels) [reply starter-bots/README.md]: Updated

**Pull request merged**